command/base.go

```diff
@@ -219,44 +218,40 @@ func (c *BaseCommand) DefaultWrappingLookupFunc(operation, path string) string {
 	return api.DefaultWrappingLookupFunc(operation, path)
 }
 
-func (c *BaseCommand) isInteractiveEnabled(mfaConstraintLen int) bool {
-	if mfaConstraintLen != 1 || !isatty.IsTerminal(os.Stdin.Fd()) {
-		return false
+// getInteractiveMFAMethodInfo returns MFA method information only if operating
+// in interactive mode and one MFA method is configured.
+func (c *BaseCommand) getInteractiveMFAMethodInfo(secret *api.Secret) *MFAMethodInfo {
+	if secret != nil && secret.Auth != nil && secret.Auth.MFARequirement != nil {
```

Comment:
I think if this logic stands, we actually want to proceed to examining MFAConstraints, and not return nil! What do you think?

Reply:
Yup, sorry, I forgot to flip the logic for the early bail-out.